keras_nlp/tokenizers/byte_pair_tokenizer.py

```diff
@@ -264,7 +267,7 @@ def __init__(
         merges,
         sequence_length=None,
         add_prefix_space=False,
-        special_tokens=None,
+        special_tokens_lst=None,
```

Comment:
I think a rename would make sense, but maybe we just name this `unsplittable_tokens` or something like that. That would be more literal as to what this is doing. Docstring...

unsplittable_tokens: A list of strings that will never be split during the word-level splitting applied before the byte-pair encoding. This can be used to ensure string special tokens map to a unique
index in the vocabulary, even if these special tokens contain splittable characters such as punctation.